app/src/components/players/PlayerStatsTable.tsx

```diff
@@ -157,10 +158,28 @@ export function PlayerStatsTable(props: PlayerStatsTableProps) {
   );
 }
 
+function getVirtualTotalLevel(skills: MapOf<Skill, SkillValue>): number {
+  let totalLevel = 0;
+  for (const [_, value] of Object.entries(skills)) {
+    if (value.metric === 'overall') continue;
+
+    totalLevel += getLevel(value.experience, true);
+  }
+
+  return totalLevel;
+}
+
+function getSkillMetrics(skills: MapOf<Skill, SkillValue>, showVirtualLevels = false): number {
+   return showVirtualLevels ? getVirtualTotalLevel(skills) : skills.overall.level;
+}
+
 function PlayerSkillsTable(
   props: PropsWithChildren<{ player: PlayerDetails; showVirtualLevels: boolean }>
 ) {
   const { children, player, showVirtualLevels } = props;
+  const skillValues = Object.values(player.latestSnapshot.data.skills)
+  .map(v => (showVirtualLevels && v.metric === Skill.OVERALL) 
+    ? { ...v, level: getSkillMetrics(player.latestSnapshot.data.skills, true)} : v);
```

Comment:
You can just call `getVirtualTotalLevel` directly here instead of `getSkillMetrics` as that's the only thing `getSkillMetrics` seems to be doing anyway. And `getSkillMetrics` can be removed.